dags/mats_dag.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+default_args = {
+    "owner": "Mats",
+    "retries": 2,
+    "retry_delay": datetime.timedelta(minutes=1),
+    "start_date": datetime.datetime(2023, 1, 2),
+}
+
+def print_hello():
+    print("Hello guys")
+
+with DAG(
+    "mats_dag",
+    default_args=default_args,
+    schedule_interval="@daily",
+) as dag:
+
+    print_hello = PythonOperator(
```

Comment:
Vet ikke om det er lurt å navngi denne variabelen det samme som det funksjonen heter, og som du henter to linjer lenger ned?